stub.py

```diff
@@ -0,0 +1,94 @@
+"""Stub class"""
```

Comment:
where should `stub.py` live? I think it makes sense to put it in `web/` somewhere. I think it would also be nice to separate the MC-controlled files from the EM-controlled files. Perhaps we can separate them into two directories in `web` and make sure that the EM only imports from MC.

Reply:
MC : tidy3d/web/core and tidy3d/web/cli
EM:  tidy3d/web excludes of tidy3d/web/core and tidy3d/web/cli

Reply:
MC : tidy3d/web/core and tidy3d/web/cli
EM: tidy3d/web/api